src/Summary.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+const Summary = ({ calculationsTable }) => {
+  console.log(calculationsTable[1]);
+  return (
+    <div className="row summary">
+      <div className="col">
```

Comment:
If we were able to dynamically compute the ranges based on table length I think we could make this more succinct.